code/src/App.js

```diff
@@ -1,9 +1,138 @@
-import React from 'react'
+import React, { useState } from "react"
+
+import "./style.css"
+import { Summary } from "./components/Summary.js"
+import { Header } from "./components/Header.js"
 
 export const App = () => {
+  const [reflection, setReflection] = useState(""); // the thing we have in the paranthesis is the initial state. If we console log status it's gonna show the thing in the paranthesis. If it's gonna show strings you use "" if it's gonna show bolean you write true or false etc, if i's gonna show an array you write []. 
+  const [reason, setReason] = useState("");
+  const [weather, setWeather] = useState("");
+  const [culture, setCulture] = useState ("");
+  const [showSummary, setShowSummary] = useState (false);
+
+  const handleSubmit = event => {
```

Comment:
This is a great little touch, preventing submit when all questions are not answered. 🙌 A minor improvement would be adding a pop-up message so that the user knows they have forgotten a question up there somewhere. :)